examples/step-76/step-76.cc

```diff
@@ -510,7 +510,7 @@ namespace Euler_DG
   EulerOperator<dim, degree, n_points_1d>::EulerOperator(TimerOutput &timer)
     : timer(timer)
   {
-#if DEAL_II_MPI_VERSION_GTE(3, 0)
+#if DEAL_II_WITH_MPI
```

Comment:
This fails on a whole bunch of builds. It needs to be like this:
```suggestion
#ifdef DEAL_II_WITH_MPI
```

Reply:
oops, of course. I was too quick...